tensorboard/webapp/metrics/views/card_renderer/scalar_card_line_chart_test.ts

```diff
@@ -1589,6 +1591,66 @@ describe('scalar card line chart', () => {
         expect(fobController.endFobWrapper).toBeUndefined();
       }));
     });
+
+    it('does not allow for single fob removal when allowFobRemoval is false', fakeAsync(() => {
```

Comment:
I think a better description is to reflect the fact, for example, what does "allow" means? Actually you mean "render".
So this one could be 'does not render dismiss icon for single fob when allowFobRemoval is false'
and other tests could be "renders..."

Reply:
Make sense. Thanks!